test/e2e-go/features/stateproofs/stateproofs_test.go

```diff
@@ -226,6 +228,80 @@ func TestStateProofOverlappingKeys(t *testing.T) {
 	r.Equalf(int(consensusParams.StateProofInterval*expectedNumberOfStateProofs), int(lastStateProofBlock.Round()), "the expected last state proof block wasn't the one that was observed")
 }
 
+func TestBlkMerkleProof(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	defer fixtures.ShutdownSynchronizedTest(t)
+
+	r := require.New(fixtures.SynchronizedTest(t))
+
+	configurableConsensus := make(config.ConsensusProtocols)
+	consensusVersion := protocol.ConsensusVersion("test-fast-stateproofs")
```

Comment:
For refactor - I'm thinking of creating a default consensusParams for state proof tests. @algoidan

Reply:
done, I've conserved other tests' different parameter usage when applicable